File: java/bank-simulation/src/main/java/com/example/bank_simulation/features/customer/CustomerController.java
package com.example.bank_simulation.features.customer;
import org.springframework.web.bind.annotation.*;
import java.util.*;

@RestController
@RequestMapping("/api/v1/customer")
public class CustomerController {
    @PostMapping("/send")
    public Map<String, String> sendMoney() {
        return Map.of("message", "Send money");
    }

    @PostMapping("/withdrawal")
    public Map<String, String> withdrawalMoney() {
        return Map.of("message", "Withdrawal money");
    }

    @GetMapping("/transaction")
    public Map<String, String> viewMyTransactions() {
        return Map.of("message", "View my transaction");
    }
}

File: dotnet/bank-simulation/Features/Customer/CustomerController.cs
using Microsoft.AspNetCore.Mvc;

namespace bank_simulation.Features.Customer;

[ApiController]
[Route("api/v1/[controller]")]
public class CustomerController : ControllerBase
{
    [HttpPost("send")]
    public object SendMoney()
    {
        return new { message = "Send money" };
    }

    [HttpPost("withdrawal")]
    public object WithdrawalMoney()
    {
        return new { message = "Withdrawal money" };
    }

    [HttpGet("transaction")]
    public object ViewMyTransactions()
    {
        return new { message = "View my transaction" };
    }
}
